src/view/show_mode/player/scenewidget.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+"""Scene widget for scene player"""
+from PySide6.QtWidgets import QWidget, QPushButton
+
+from model import Scene
+from file.write import create_xml
+
+
+class SceneWidget(QPushButton):
```

Comment:
This is a button that saves the show file. Why is it called SceneWidget?!?

Reply:
Renamed to SceneTransmitButton